lib/extractor/index.ts

```diff
@@ -203,27 +203,25 @@ function layersWithLatestFileModifications(
     // go over extracted files products found in this layer
     for (const filename of Object.keys(layer)) {
```

Comment:
What do you think about having "if-else" chains here instead of `continues`?

How about extracting code to separate functions and using "return" instead?